daily-challenges/2024-11/04/ththao27.spec.ts

```
@@ -0,0 +1,28 @@
+function calculateMagicGate(calculations: string[]): number[]{
+    return calculations.map((calc) => {
+        const match = calc.match(/(-?\d+)\s*([\+\-\*\/])\s*(-?\d+)/);
+        if (match) {
+            const num1 = parseInt(match[1]);
```

Biome

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)